e2e/yaks/common/kamelet-binding-http/timer-source-binding-display.yaml

```diff
@@ -15,8 +15,8 @@
 # limitations under the License.
 # ---------------------------------------------------------------------------
 
-apiVersion: camel.apache.org/v1alpha1
-kind: KameletBinding
```

Comment:
Is this the only KameletBinding we still have? we have an open issue to remove them all https://github.com/apache/camel-k/issues/5490 - feel free to tackle into this same PR if you want.